openshift/build_parameters

```diff
@@ -0,0 +1,7 @@
+APPLICATION_NAME=duke-data-service
+GIT_REPO_URL=https://github.com/dmlond/duke-data-service.git
```

Comment:
s/dmlond/Duke-Translational-Bioinformatics/?

Reply:
I can do that, but this is really just an example to be modified as needed for testing. For example, the GIT_REF and APPLICATION_ENVIRONMENT would almost invariably need to be changed during manual testing for different git branches/tags